docs/docs/versions/v0_3/index.mdx

```diff
@@ -205,3 +207,48 @@ class FooParser(BaseOutputParser):
 
 FooParser.model_rebuild()
 ```
+
+
+## Migrate using langchain-cli
+
+The `langchain-cli` can help migrate your code to new imports automatically. As of 0.0.31, the `langchain-cli` relies on [gritql](https://about.grit.io/) for applying code mods.
+
+## Installation
+
+```bash
+pip install langchain-cli
+langchain-cli --version # <-- Make sure the version is at least 0.0.31
+```
+
+## Usage
+
+Given that the migration script is not perfect, you should make sure you have a backup of your code first (e.g., using version control like `git`).
+
+You will need to run the migration script **twice** as it only applies one import replacement per run.
```

Comment:
should we just run it twice for users?

Reply:
Not enough of a friction point to do this I think